include/RED4ext/CNamePool-inl.hpp

```diff
@@ -48,3 +48,134 @@ RED4EXT_INLINE const char* RED4ext::CNamePool::Get(const CName& aName)
 
     return "";
 }
+
+RED4EXT_INLINE RED4ext::CNamePool* RED4ext::CNamePool::GetPool()
+{
+    static UniversalRelocFunc<CNamePool* (*)()> func(Detail::AddressHashes::CNamePool_GetPool);
+    return func();
+}
+
+RED4EXT_INLINE RED4ext::CNamePoolNode* RED4ext::CNamePoolNodeInner::Outer()
+{
+    return reinterpret_cast<RED4ext::CNamePoolNode*>(reinterpret_cast<uintptr_t>(this) - offsetof(RED4ext::CNamePoolNode, inner));
```

Comment:
You can omit `RED4ext::` here and in similar cases.
And you may want to run clang-format for all files, ideally set up [pre-commit hook](https://github.com/wopss/RED4ext.SDK/blob/master/CONTRIBUTING.md#general).